src/order/rel_classes.lean

```diff
@@ -301,6 +301,13 @@ theorem not_lt_min {α} {r : α → α → Prop} (H : well_founded r)
   (p : set α) (h : p.nonempty) {x} (xp : x ∈ p) : ¬ r x (H.min p h) :=
 let ⟨_, h'⟩ := classical.some_spec (H.has_min p h) in h' _ xp
 
+theorem has_min' [partial_order α] : (well_founded (gt : α → α → Prop)) →
```

Comment:
This definition is should be called `has_max` because we get the largest element of the set `a` instead of the smallest. I think we should have both `has_min` (assuming `(<)` is well-founded) and `has_max` (assuming `(>)` is well-founded.